hazelcast/src/test/java/com/hazelcast/client/console/ClientConsoleAppTest.java

```diff
@@ -185,12 +186,7 @@ private void assertTextInSystemOut(String substring) {
      * Gets content of standard output buffer.
      */
     private static String getSystemOut() {
-        try {
-            return new String(baos.toByteArray(), "UTF-8");
-        } catch (UnsupportedEncodingException e) {
-            //This should never happen. Everybody loves and supports the UTF-8
-            throw new AssertionError("Get a deep breath and continue with reading. Your Java doesn't support UTF-8.");
-        }
```

Comment:
The nerdy pun needs to go away since the constructor accepting `Charset` object doesn't throw an exception.